recipe/meta.yaml

```diff
@@ -11,23 +11,20 @@ source:
   sha256: e4e1d9366fc4ca60b4b2354b6d12e65600600a8c7bf4392c84f2f4ff4abc85ff
 
 build:
-  number: 2
-  skip: true  # [win or py<36]
+  number: 3
+  noarch: python
   script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
-  build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
   host:
     - pip
-    - python
+    - python >=3.6
   run:
     - docutils >=0.3
     - psutil
-    - python
+    - python >=3.6
     - setuptools
-
+    - __unix
```

Comment:
This virtual package won't be around on OSX or windows. Thus this build won't install on those systems. Is that intended? I could not tell from the comment thread.

Reply:
In particular, this statement is not true

> Hey, yes that's fine and pynisher still be available on macOS.

Reply:
? `__unix` implies Linux and macOS

Reply:
ack! I did not read that properly 🤦 

I read linux not unix. Sorry!